utilities/provider_tallies/README.md

```diff
@@ -8,8 +8,8 @@ localhost (usually via tunneling).
 
 ## Usage
 
-1. `pipenv install`
-2. `pipenv run python provider_tally_stats.py`
+1. `../../ov pdm install`
+2. `../../ov pdm run provider_tally_stats.py`
```

Comment:
Thanks to #4567 and how `ov` is now aware of working directories, this can just be:
```suggestion
1. `ov pdm install`
2. `ov pdm run provider_tally_stats.py`
```